nullaway/src/main/java/com/uber/nullaway/autofix/AutoFixConfig.java

```diff
@@ -0,0 +1,351 @@
+package com.uber.nullaway.autofix;
+
+import com.google.common.base.Preconditions;
+import com.sun.source.util.Trees;
+import com.sun.tools.javac.code.Symbol;
+import com.uber.nullaway.autofix.qual.AnnotationFactory;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import javax.lang.model.element.Element;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+
+public class AutoFixConfig {
+
+  public final boolean MAKE_METHOD_TREE_INHERITANCE_ENABLED;
```

Comment:
All uppercase field names imply hard-coded constants, which implies both that these should be `static final` and that their value should be assigned only once at class initialization (and ideally inline with their definition). 

If these are actually input-determined configuration, then they should be normal camel-case fields as with `AbstractConfig`.

Also, are all these levers for the algorithm still used? Or are there unchanging parameters these days? Asking because, eventually, we would need to document every one of these configuration options and the configuration file format...

Reply:
Thank you for the note, they are input determined and I converted them all to camel case.